src/apps/schema/src/app/views/SchemaEdit/FieldSettings/FieldSettings.js

```diff
@@ -71,11 +71,12 @@ export default function FieldSettings(props) {
             className={styles.Setting}
             name="label"
             label="Field Label"
-            {...(props.new
```

Comment:
@agalin920 please provide a brief explanation of why this should remain as is
@rjpals please include @agalin920 explanation as a comment for future reference

Reply:
@rjpals If this field is not a new field it has an existing value. If said existing value gets passed as a 'value' prop to the input then upon editing it will perform a cursor jump (we will eventually have components that avoid this behavior). By setting an existing value as a defaultValue this initial cursor jump is avoided